README.md

```diff
@@ -92,3 +92,17 @@ The config setting `COG_THREADS` controls how many threads should be requested w
 out COGs. Ideally this will be set to at least 2 (preferably 4).
 Higher values do seem to reduce write times but with diminishing returns (tested up to 8).
 Locally, write times with four threads configured range from 10 to 15 seconds.
+
+## Reef edge alignment for site searching
```

Comment:
I was of two minds about this being in the readme, but we can keep it here until we have a proper documentation site up

Reply:
Yeah, do you think these requirements should be put into a sort of quick preprocessing function. So someone puts in their gdf, raster pixels and we return the reef_lines, search_pixels and indices for function input. That might make it more streamlined for the APi?